lib/average_end_of_day_depository_balance/main.py

```diff
@@ -68,17 +68,18 @@ async def get_average_end_of_day_depository_balance(
             balance_dict["institution_id"] = institution_id
 
             record_list.append(balance_dict)
-    
-    # if no data is present, consider the sum of balances to be non-existing 
+
+    # if no data is present, consider the sum of balances to be non-existing
     if len(record_list) == 0:
         return None
 
     # STEP 5: convert to pandas dataframe to help with the resampling
     balances_df = pd.DataFrame(record_list)
 
     # Sort and create a column for day, filter by time window
-    balances_df = balances_df.sort_values("ts")
-    balances_df["yyyymmdd"] = pd.to_datetime(balances_df["ts"], unit="s")
+    balances_df["timestamp"] = balances_df["timestamp"].astype('datetime64[D]')
```

Comment:
What's the benefit to using `.astype('datetime64[D]')` here versus `pd.to_datetime(...)`?

Reply:
Because of the new format of how the `timestamp` is returned from the response, we won't be able to specify `unit="s"` anymore to `to_datetime()`. I remember reading that without specifying the `unit` value, the difference in performance is quite significant compared to using `astype()` ([`astype()` being better](https://github.com/pandas-dev/pandas/issues/17976), even though this talks about the value being passed being an epoch timestamp, which ain't true in our case).

I don't feel strong about using either of the above ^

Reply:
Performance differences should be relatively minor, so I'd be in favor of being consistent with the approach between the two.